### PR TITLE
Translate Ruby 2.4.7 released news (id)

### PR DESCRIPTION
**Pull request by meisyal**

### CODE REVIEW

File: id/news/_posts/2019-08-28-ruby-2-4-7-released.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+layout: news_post
+title: "Ruby 2.4.7 Rilis"
```
Comment (kuntoaji) [inline id/news/_posts/2019-08-28-ruby-2-4-7-released.md]: I prefer "Ruby 2.4.7 Dirilis" or "Ruby 2.4.7 Telah Dirilis" for the title

File: id/news/_posts/2019-08-28-ruby-2-4-7-released.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+Ruby 2.4.7 telah dirilis.
+
+Rilis ini mencakup sebuah perbaikan keamanan.
+Mohon cek topik-topik di bawah ini untuk detail.
+
+* [Beberapa kerentanan jQuery pada RDoc](/id/news/2019/08/28/multiple-jquery-vulnerabilities-in-rdoc/)
+
+Ruby 2.4 saat ini berada pada fase perawatan keamanan hingga akhir Maret 2020.
```
Comment (kuntoaji) [inline id/news/_posts/2019-08-28-ruby-2-4-7-released.md]: I think "pemeliharaan" is better than "perawatan"

File: id/news/_posts/2019-08-28-ruby-2-4-7-released.md (hunk 1 above)
Comment (meisyal) [reply id/news/_posts/2019-08-28-ruby-2-4-7-released.md]: OK. I just updated that to "Ruby 2.4.7 Telah Dirilis".

By the way, how about the old translations that have been production? Should we update it? or just ignore it and focus on the next translation? @kuntoaji

File: id/news/_posts/2019-08-28-ruby-2-4-7-released.md (hunk 2 above)
Comment (meisyal) [reply id/news/_posts/2019-08-28-ruby-2-4-7-released.md]: This one also. Could you check it again, @kuntoaji?

File: id/news/_posts/2019-08-28-ruby-2-4-7-released.md (hunk 1 above)
Comment (kuntoaji) [reply id/news/_posts/2019-08-28-ruby-2-4-7-released.md]: I think it's better focus on new translation first for now.

Comment (meisyal) [reply id/news/_posts/2019-08-28-ruby-2-4-7-released.md]: OK. Can I merge this now, @kuntoaji?

**Comment by meisyal** [comment]
Hi @kuntoaji, do you still have a comment for this translation?

Thank you.

File: id/news/_posts/2019-08-28-ruby-2-4-7-released.md (hunk 1 above)
Comment (kuntoaji) [reply id/news/_posts/2019-08-28-ruby-2-4-7-released.md]: yup

**Pull request merged**